src/SFA.DAS.EmployerFeedback.Web/Views/Shared/_CookieConsentPartial.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@using Microsoft.AspNetCore.Http.Features
```

Comment:
Please double-check if this partial view is being used. I’m not sure if this is dynamically referenced. If not, consider removing.